packages/vue/src/composables/form.ts

```diff
@@ -177,7 +177,7 @@ export function useForm<T extends Fields = Fields>(options: FormOptions<T>) {
 	/**
 	 * Clears all errors.
 	 */
-	function clearErrors(...keys: (keyof T)[]) {
+	function clearErrors(...keys: string[]) {
```

Comment:
Do you think we can have auto-completion there using `@clickbar/dot-diver` like [`useProperty`](https://github.com/hybridly/hybridly/blob/0.x/packages/vue/src/composables/property.ts)?

Reply:
I'll have a look, but I agree it would be neat.